### DISCUSSION TITLE
Renaming and Deleting insights Dashboards

### DISCUSSION THREAD

**Discussion by harshadsatra** [directus/directus#11875] [Archive]
Earlier, I created an insight dashboard with the name *Demo* to test out this feature, and later when I wanted to delete this or rename this, there is no such option. I had to go to the database and rename it there to *Dashboard* as you can see in the screenshot below.

<img width="1379" alt="Screenshot 2022-02-27 at 11 35 14 AM" src="https://user-images.githubusercontent.com/5407138/155870435-9517c471-80e4-4ef6-99da-e4db99f3ee83.png">

**Comment by abdonrd** [comment]
You can do it from the Insights home page:

<img width="894" alt="Screenshot 2022-02-27 at 20 44 10" src="https://user-images.githubusercontent.com/1007051/155897320-0fa5e3eb-76c3-4bd6-ab06-7972e1266824.png">
